src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -110,6 +110,21 @@ const getAuthenticatorData = function (authenticator, isVerifyAuthenticator) {
     });
     break;
   }
+
+  case AUTHENTICATOR_KEY.IDP: {
+    const idpName =  authenticator.relatesTo?.displayName;
+    const factorDescription = idpName
+      ? loc('oie.customFactor.authenticator.description', 'login', [idpName])
+      : loc('oie.customFactor.authenticator.description.default', 'login');
+    Object.assign(authenticatorData, {
+      description: isVerifyAuthenticator
+        ? ''
+        : factorDescription,
+      iconClassName: 'mfa-custom-factor',
```

Comment:
nit: maybe this can be also `mfa-idp-authenticator`?

Reply:
This will require CSS-level changes to extend the existing `mfa-custom-factor` icon to `mfa-idp-authenticator`.

Since this value is internal only and will not be read via a screen reader - I think we should keep this as-is to avoid the extra overrides. That work?